app/javascript/dashboard/show.js

```diff
@@ -117,6 +111,39 @@ OWSO.DashboardShow = (() => {
     );
   }
 
+  function updateChartOption(chart) {
+    let max = _.max(flatten(chart.data.datasets));
+    let padding = chart.config.type == "horizontalBar" ? 1.75 : 1.4;
+    let suggestedMax = Math.round(max * padding);
+    let labelsLength = 0,
+      maxRotation = 0,
+      minRotation = 0;
+
+    let yTicks = chart.options.scales.yAxes[0].ticks;
+    let xTicks = chart.options.scales.xAxes[0].ticks;
+
+    if (
+      chart.canvas.id.match(
+        /^chart_owso_feedback_trend|chart_information_access_by_period/
+      )
+    ) {
+      labelsLength = chart.data.labels.length;
+      maxRotation = minRotation = labelsLength < 7 ? 0 : 45;
```

Comment:
@radinreth consider moving 7 into a variable with a giving meaningful name so that we easily reflect that later on

Reply:
updated bong